src/components/Home.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable linebreak-style */
+import React from 'react';
+
+export default function Home() {
+  return (
+    <>
+      <main className="home">
+        <h2>Welcome to the homepage!</h2>
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
+          incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud
+          exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure
+          dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
+          Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt
+          mollit anim id est laborum.
+        </p>
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
+          incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud
+          exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure
+          dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
+          Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt
+          mollit anim id est laborum.
+        </p>
+      </main>
```

Comment:
[REQUIRED] On the first load, the homepage is usually blank before refreshing or navigating. Kindly ensure that the text on the homepage is displayed without the user carrying out any action. This seems to happen because you have an extra `/math-magicians` endpoint:

![image](https://user-images.githubusercontent.com/62938735/208127172-4301bc61-bc96-49d3-ac6c-c808e317afc9.png)

Reply:
Thank you, sir, for raising my attention to this point.